src/test/java/Homework16.java

```diff
@@ -0,0 +1,36 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import java.time.Duration;
+public class Homework16 extends BaseTest {
+
+    
+@Test
+
+public void registrationNavigation() { {
+
+    ChromeOptions options = new ChromeOptions();
+    options.addArguments("--remote-allow-origins=*");
+    options.addArguments("--start-maximized");
+
+    WebDriver driver = new ChromeDriver(options);
+    driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+    String url = "https://qa.koel.app/";
+    driver.get(url);
+
+    WebElement registrationLink = driver.findElement(By.cssSelector("//a[@href='registration']"));
+    registrationLink.click();
+
+    Assert.assertEquals(driver.getCurrentUrl(), url);
```

Comment:
When use click registrationLink link and page navigate to registration page then now the url will change, no longer `url = "https://qa.koel.app/"; `
Please update it